lib/trento/discovery/payloads/host_discovery_payload.ex

```diff
@@ -18,6 +18,7 @@ defmodule Trento.Discovery.Payloads.HostDiscoveryPayload do
   deftype do
     field :hostname, :string
     field :ip_addresses, {:array, :string}
+    field :ip_addresses_netmasks, {:array, :string}
```

Comment:
I'm still not entirely convinced of having a 2nd field that sends "duplicated" data.
What if have this logic in the policy.
You just check if the coming address has the mask to remove it for the `ip_addresses` field, and leave the `ip_addresses_netmask` as is in the read model.
PD: I'm not saying to not have a new entry in the read model. It is only for the payload itself